db/column_family.cc

```diff
@@ -616,7 +619,7 @@ void ColumnFamilyData::RecalculateWriteStallConditions(
                imm()->NumNotFlushed() >=
                    mutable_cf_options.max_write_buffer_number - 1) {
       write_controller_token_ =
-          SetupDelay(ioptions_.delayed_write_rate, write_controller,
+          SetupDelay(mutable_db_options_.delayed_write_rate, write_controller,
```

Comment:
`ColumnFamilyData` and `mutable_cf_options` are specific to one column family. Each time we change `mutable_cf_options`, we update the struct for the specific column family. `mutable_db_options` is global to the whole DB. While it is possible, I don't like the idea to have to update all column family when we change it.